install/macos.md

```diff
@@ -60,29 +69,35 @@ __Wichtig__: Wir wollen Python 3.8 Graphical Installer (unten auf der Seite).  D
 
 - [VS Codium](https://github.com/VSCodium/vscodium/releases): Texteditor.
    Download der Datei `VsCodium-darwin-<VERSION>.zip`.
-   Das Programm "VSCodium.app" befindet sich dann in der zip Datei im Downloadordner und muss nach `Programme` verschoben werden. Die App öffnen und mit `Shift+cmd+p` die Kommando Palette öffnen und `Shell Command: Install 'code' command in PATH` ausführen. Nun wird VS Code auch geöffnet, wenn im Terminal "code" eingeben wird.
+   Das Programm "VSCodium.app" befindet sich dann in der zip Datei im Downloadordner und muss nach `Programme` verschoben werden. Die App öffnen und mit `Shift+cmd+p` die Kommando Palette öffnen und `Shell Command: Install 'codium' command in PATH` ausführen. Nun wird VS Codium auch geöffnet, wenn im Terminal "codium" eingeben wird.
 
 Um git beizubringen, VSCodium zu benutzen, Im Terminal:
 
 ```bash
 git config --global core.editor "codium --wait"
 ```
 
-- Damit VS Code LaTeX-Code besser darstellen kann, muss das Plugin `LaTeX language support`
-  installiert werden
+- Damit VS Code LaTeX-Code besser darstellen kann, muss das Plugin `LaTeX Workshop`
+  installiert werden:
     - Mit `Shift+cmd+x` die Extensions öffnen
-    - `LaTeX language support` suchen
+    - `LaTeX Workshop` suchen
     - `install` anklicken
-    - VS Code neustarten
+    - VS Codium neustarten
+
+- Um LaTex Workshop für für uns richtig einzustellen:
+    - Settings aufrufen `File -> Preferences -> Settings`
+    - Hacken bei `latex auto build clean and retry` entfernen
+    - Bei `latex auto build run` im Drop-Down-Menü `never` aufwählen
+
 
 ### Uncertainties
 
 - [Uncertainties](http://pythonhosted.org/uncertainties/): Python-Bibliothek
   für automatisierte Fehlerrechnung
 
-Im terminal:
+Im Terminal:
 
-    pip install uncertainties
+    pip3 install uncertainties
```

Comment:
no. The correct form is either `python3 -m pip` which is the general best practive, or just `pip` when relying on that anaconda is first in path. `pip3` doesn't make sense.